src/main/resources/io/jenkins/plugins/pipelinegraphview/Messages.properties

```diff
@@ -0,0 +1,24 @@
+# The MIT License
```

Comment:
All of these files have been copied over from core hence the licensing information at the top, this can either be unified, retained, or altered in some other way as necessary